libraries/AP_Compass/CompassCalibrator.h

```diff
@@ -22,7 +22,7 @@ class CompassCalibrator {
 
     CompassCalibrator();
 
-    void start(bool retry=false, float delay=0.0f);
+    void start(bool retry=false, float delay=0.0f, uint16_t offset_max=600);
```

Comment:
Why the default value? I think the parameter should always be used, so having a default doesn't look right to me.